test/reporter/report.js

```diff
@@ -7,7 +7,7 @@ if( !process.env.SUITE ) {
     return;
 }
 
-const filePath = '../log/selenium/result-'+ process.env.SUITE + '.json';
+const filePath = `../log/${process.env.SUITE}/selenium-result.json`;
```

Comment:
This path actually gets overridden by the WDIO JSON reporter to `log/$SUITE/result.json`. This was the case before they branch, and this may be best removed here or otherwise commented for clarity.